docs/BR.md

```diff
@@ -3442,6 +3476,13 @@ If the signing key is P-384, the signature MUST use ECDSA with SHA-384. When enc
 
 If the signing key is P-521, the signature MUST use ECDSA with SHA-512. When encoded, the `AlgorithmIdentifier` MUST be byte-for-byte identical with the following hex-encoded bytes: `300a06082a8648ce3d040304`.
 
+##### 7.1.3.2.4 ML-DSA
+
+The CA SHALL use the appropriate signature algorithm and encoding based upon the signing key used.
+
+If the signing key is ML-DSA-87, the signature algorithm SHALL be id-ml-dsa-87 (OID: 2.16.840.1.101.3.4.3.19). When encoded, the `AlgorithmIdentifier` SHALL be byte-for-byte identical with the following hex-encoded bytes: `300b0609608648016503040313`.
+
```

Comment:
Since we're expecting only one relying party platform to support ML-DSA-87 in serverAuth certificates in the short term, I suggest adding text (perhaps with a note about it being a temporary requirement/advice) to require or recommend that ML-DSA-87 private keys must/should only sign ML-DSA-87 certificates.

This would help to ensure no interop issues with existing publicly-trusted RSA and ECC WebPKI hierarchies, and (I'm speculating) might help to make this ballot more palatable to some Root Store voting members.

Reply:
Fix for the above pushed.